src/DashboardHub/Bundle/AppBundle/Tests/Spec/DashboardHub/Bundle/AppBundle/Service/DashboardServiceSpec.php

```diff
@@ -392,4 +393,23 @@ function it_should_find_all_by_is_public_and_popular(
 
         $this->findAllByIsPublicAndPopular();
     }
+
+    function it_should_search(
+        EntityManager $em,
+        SecurityContext $securityContext,
+        DashboardRepository $dashboardRepository,
+        Search $search
+    )
+    {
+        $dashboardRepository->search($search)
```

Comment:
Hmm, why? I have always seen the first method call to be on the same line as the initial variable.

Or have you got used to do that because of PHPUnit code coverage issue?